core/modules/simpletest/backdrop_web_test_case.php

```diff
@@ -1347,8 +1347,9 @@ protected function backdropCreateRole(array $permissions, $name = NULL) {
       $name = $this->randomName();
     }
 
-    // Check the all the permissions strings are valid.
-    if (!$this->checkPermissions($permissions)) {
+    // Check that all the permission strings are valid.
+    // @todo: Remove this change when we sort out https://github.com/backdrop/backdrop-issues/issues/5354
```

Comment:
Let's be more specific so we don't accidentally delete the whole check...
`// @todo: Remove the TRUE parameter`